Mod2/Lessons/Week2/ProductiveStruggle.md

```diff
@@ -0,0 +1,53 @@
+# Productive Struggle
+
+### Warm Up
+Reflect in your notebook on the following:
+* During the first few days of the Mod, when did you feel like you were struggling the most?
```

Comment:
I wonder if it would be better to have folks reflect on their whole time at turning instead of only week 1 of Mod 2.

Reply:
Yes, agreed!